product_barcode_generator/models/product.py

```diff
@@ -30,67 +30,67 @@ def isodd(x):
 class ProductCategory(models.Model):
     _inherit = 'product.category'
 
-    ean_sequence_id = fields.Many2one('ir.sequence', string='Ean sequence')
+    barcode_sequence_id = fields.Many2one(comodel_name='ir.sequence',
+                                          string='Ean sequence')
```

Comment:
We may also want to change the string attribute if we're abstracting the definition of barcode